packages/loader/container-loader/api-report/container-loader.legacy.alpha.api.md

```diff
@@ -151,6 +178,12 @@ export type QuorumProposalsSnapshot = {
     values: [string, ICommittedProposal][];
 };
 
+// @alpha
+export function rehydrateDetachedContainerFromSnapshot(rehydrateDetachedContainerProps: ILoaderRehydrateDetachedContainerProps): Promise<IContainer>;
```

Comment:
I think we can just simplify this to `rehydrateDetachedContainer`, and the property in the rehydrateDetachedContainerProps specifies what we are rehyrdating from